<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectWidgetClass</class>
 <widget class="QWidget" name="ConnectWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ConnectWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label_dbname">
     <property name="text">
      <string>DatabaseName:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_dbname"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_username">
     <property name="text">
      <string>UserName:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_username"/>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_password">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="QLineEdit" name="lineEdit_password"/>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>69</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_certain">
     <property name="text">
      <string>certain</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>84</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="4">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>59</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
